core/templates/pages/exploration-player-page/services/audio-translation-manager.service.ts

```diff
@@ -32,16 +32,16 @@ export interface AudioTranslations {
 })
 export class AudioTranslationManagerService {
   // Audio translations for the main content of a card at the top.
-  _contentAudioTranslations: AudioTranslations = null;
-  _contentHtmlForAutogeneratedAudio: string = null;
+  _contentAudioTranslations: AudioTranslations | null = null;
+  _contentHtmlForAutogeneratedAudio: string | null = null;
 
   // Audio translations for everything other than the main content, such as
   // those for feedback, hints, and solutions.
-  _secondaryAudioTranslations: AudioTranslations = null;
-  _secondaryHtmlForAutogeneratedAudio: string = null;
+  _secondaryAudioTranslations: AudioTranslations | null = null;
+  _secondaryHtmlForAutogeneratedAudio: string | null = null;
 
-  _currentPrimaryComponentName: string = null;
-  _currentSecondaryComponentName: string = null;
+  _currentPrimaryComponentName: string | null = null;
+  _currentSecondaryComponentName: string | null = null;
```

Comment:
Not sure if we want to allow `null` in these, is there other way on how to do this? Think about the context where `AudioTranslationManagerService` is used and if we need these to be `null` in the usage places.

Reply:
@vojtechjelinek Thanks for reviewing. Null is being used as a default value here.
Can't we just use the default values in the case that you mentioned.
And since the return value of the get functions has type `getCurrentAudioTranslations(): AudioTranslations | null`
I dont think there should be a problem. PTAL!

Reply:
@vojtechjelinek Thanks for the review.
Would using undefined be a better option?

Reply:
No, it would be best to disallow both `null` and `undefined`.

Reply:
Looking at the code I have no idea how to refactor this though, so probably fine to add the `null` for now.